docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,199 @@
+NOTE TO REMOVE: Due to problems with exporing, the training was launched on https://github.com/Samsung/ONE/pull/13226
+```bash
+onert_train \
+--modelfile customized_mobilenetv2.circle \
+--epoch 5 \
+--loss 1 \                # mean squared error
+--loss_reduction_type 1 \ # sum over batch size
+--optimizer 2 \           # adam
+--learning_rate 0.0001 \
+--batch_size 32 \
+--num_of_trainable_ops 5 \
+--load_expected:raw cats_and_dogs.output.bin \
+--load_input:raw cats_and_dogs.input.bin \
+--export_path customized_mobilenetv2_trained.circle
+```
+The result of training:
+```
+Model Expected Filename cats_and_dogs.output.bin
+Model Input Filename cats_and_dogs.input.bin
+Model Filename customized_mobilenetv2.circle
+== training parameter ==
+- learning_rate        = 0.0001
+- batch_size           = 32
+- loss_info            = {loss = mean squared error, reduction = sum over batch size}
+- optimizer            = adam
+- num_of_trainable_ops = 10
+========================
+Epoch 1/5 - time: 532.378ms/step - loss: [0] 0.1567
+Epoch 2/5 - time: 531.966ms/step - loss: [0] 0.0417
+Epoch 3/5 - time: 532.795ms/step - loss: [0] 0.0104
+Epoch 4/5 - time: 532.607ms/step - loss: [0] 0.0053
+Epoch 5/5 - time: 532.567ms/step - loss: [0] 0.0049
+===================================
+MODEL_LOAD   takes 4.3720 ms
+PREPARE      takes 192.9690 ms
+EXECUTE      takes 165537.7600 ms
+- Epoch 1      takes 33007.4380 ms
+- Epoch 2      takes 32981.8970 ms
+- Epoch 3      takes 33033.2690 ms
+- Epoch 4      takes 33021.6380 ms
+- Epoch 5      takes 33019.1300 ms
+===================================
+```
+## Test inference
+In this tutorial the inference will be launched via `Python API`.
+### Convert the trained model to nnpackage format
+The current version of `Python API` expect models in Python nnpackge format. The conversion can be done with `tools/nnpackage_tool/model2nnpkg/model2nnpkg.py` script.
+```bash
+python model2nnpkg.py -m customized_mobilenetv2_trained.circle
+```
+### Run inference via Python API
+PRE: Install nnfwapi package
+```
+pip install -i https://test.pypi.org/simple/ nnfwapi==0.1.1
+```
+```python
+from nnfwapi.libnnfw_api_pybind import *
+
+import matplotlib.pyplot as plt
+import os
+import tensorflow as tf
+
+nnpackage_path = '/home/m.bencer/workspace/OneLearningNotes/test_models/mobilenetv2/customized_mobilenetv2_trained'
+session = nnfw_session(nnpackage_path, 'cpu')
+
+_URL = 'https://storage.googleapis.com/mledu-datasets/cats_and_dogs_filtered.zip'
+path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)
+PATH = os.path.join(os.path.dirname(path_to_zip), 'cats_and_dogs_filtered')
+
+validation_dir = os.path.join(PATH, 'validation')
+
+BATCH_SIZE = 1 # we take only a single image per inference
```

Comment:
At first I've planed to inference more images (the model has dynamic batch) but [set_input_tensorinfo](https://github.com/Samsung/ONE/blob/1389776f542af795e13ad9efe03393fc252dbe69/runtime/onert/api/python/include/nnfw_api_wrapper.h#L123) causes segfault probably because it should be called before `nnfw_prepare` which is already called in [NNFW_SESSION ctor](https://github.com/Samsung/ONE/blob/1389776f542af795e13ad9efe03393fc252dbe69/runtime/onert/api/python/src/nnfw_api_wrapper.cc#L164)